src/util/helpers.tsx

```diff
@@ -248,7 +246,11 @@ export const continueOrFinish = (
 };
 
 export const logout = (): void =>
-  void fetch("/oidc/logout").then(() => window.location.reload());
+  void fetch("/oidc/logout").then(() => {
+    if (!window.location.href.includes("/ui/login")) {
```

Comment:
do we actually need to do the redirect here? Since authentication check is already happening in App now maybe we let it handle the redirect there?

Reply:
actually nvm, the user would have to perform an action before they get logged out otherwise, please disregard my previous comment

Reply:
Doing it here catches three cases:
1. A user browses to a page that requires login, in this case the App triggers this function and it redirects to login.
2. A user has the UI open on a page that requires login. They loose access (i.e. the cert gets removed server side or the OIDC cookie expires) and come back. The UI refreshes its state with API calls and they hit a 403. App will trigger this function and it redirects to login.
3. A user clicks on logout in the Navigation. We want to immediately send the user to the login page after invalidating and removing their cookie.

Reply:
Your comment brought me to realized we don't need the "did we hit a 403" check in handling any api responses (in helper.tsx) to trigger logout. Becuase if we hit a 403, App will rerender and trigger this function. Which is better, because we could have multiple api calls triggering logout multiple times. App will only trigger it once.

Reply:
Yes, it's better to handle this check in one place :+1: